account_operating_unit/models/res_company.py

```diff
@@ -0,0 +1,36 @@
+# © 2019 Eficent Business and IT Consulting Services S.L.
+# © 2019 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools.translate import _
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    inter_ou_clearing_account_id = fields.Many2one(
+        "account.account",
+        "Inter-operating unit\
+                                                   clearing account",
+    )
+
+    ou_is_self_balanced = fields.Boolean(
+        "Operating Units are self-balanced",
+        help="Activate if your company is "
+        "required to generate a balanced"
+        " balance sheet for each "
+        "operating unit.",
+    )
+
+    @api.constrains("ou_is_self_balanced")
```

Comment:
```suggestion
    @api.constrains("ou_is_self_balanced", "inter_ou_clearing_account_id")
```
IMO, If you change inter_ou_clearing_account_id to empty and ou_is_self_balanced still check. The system should show raise error too.